A2-service-configs-in-dns.md

```diff
@@ -55,9 +55,12 @@ determine which choice will be selected by a given client:
     // "python", etc).  Each string is case insensitive.
     "clientLanguage": [string],
     // Percentage: integer from 0 to 100 indicating the percentage of
-    // clients that should use this choice.  If present, the number must
-    // match the regular expression `^0|[0-9]|[1-9][0-9]|100$`
-    // All other numbers are considered invalid.
+    // clients that should use this choice.  Each client's resolver should
+    // compute its percentage at startup and use the same percentage for the
+    // lifetime of the resolver; if the client's computed percentage is less
```

Comment:
lifetime of the process?  Does this work with python multiprocessing?

Reply:
In C-core, we don't really have anything that exists for the lifetime of the process.  Even our most global data structures only exist between when grpc_init() and grpc_shutdown() are called, and that can happen multiple times over the lifetime of a process.  I suppose we could use that scope for this, but that would add complexity, and it's not clear what the advantage of that would be.  I think lifetime of the resolver (which is basically the lifetime of the channel) is fine.

I don't think this will be a problem for python or any other wrapped language, since it's purely within the channel implementation in C-core.